go.mod

```diff
@@ -6,6 +6,7 @@ require (
 	github.com/google/go-cmp v0.5.6
 	github.com/stretchr/testify v1.7.0
 	go.opentelemetry.io/otel/oteltest v1.0.0-RC1
+	go.opentelemetry.io/otel/sdk v1.0.0-RC1
```

Comment:
This is the API module and `MUST NOT` depend on any SDK.

Reply:
The API module probably MUST NOT depend on the oteltest :D

Reply:
@Aneurysm9 if you resolve https://github.com/open-telemetry/opentelemetry-go/issues/2077 then this will go away :D